src/ci/docker/scripts/x86_64-gnu-llvm.sh

```diff
@@ -6,10 +6,15 @@ set -ex
 if [[ -z "${PR_CI_JOB}" ]]; then
     # When running gcc backend tests, we need to install `libgccjit` and to not run llvm codegen
     # tests as it will fail them.
+    # The tests are run a second time with the size optimizations enabled.
     if [[ "${ENABLE_GCC_CODEGEN}" == "1" ]]; then
         ../x.py --stage 1 test --skip src/tools/tidy --skip tests/codegen
+        ../x.py --stage 1 test --skip src/tools/tidy --skip tests/codegen \
+            --rustc-args "--cfg feature=\"optimize_for_size\""
     else
         ../x.py --stage 1 test --skip src/tools/tidy
+        ../x.py --stage 1 test --skip src/tools/tidy --rustc-args \
```

Comment:
This essentially runs the whole test suite, or at least a large part of it. It would be great to just run the stdlib tests for now, I think that would be enough. I would suggest this instead:
```suggestion
        ../x.py --stage 1 test library/std library/alloc library/core --skip src/tools/tidy --rustc-args ...
```

Reply:
Yeah that makes sense. Updated it